STLYEGUIDE_SPRESS_UPDATE/src/content/elements/element-forms.html

```diff
@@ -0,0 +1,137 @@
+---
+layout: element
+section: element
+
+title: Form Elements
+description: Form elements show the basic styles for form items as they will appear on the site. This includes some basic layout options.
+
+---
+
+    <form id="myForm" action="#" method="post">
+
+      <div class="form-row">
+        <div class="input-wrapper">
+          <label for="name">Text Input:</label>
```

Comment:
@aCyborg I think it would be helpful to also include text input with the following attributes:

- read only

- required

- disabled

-